src/services/auth.service.js

```diff
@@ -0,0 +1,35 @@
+import nodemailer from 'nodemailer';
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  secure: true,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+  tls: {
+    servername: 'smtp.gmail.com',
+  },
+});
+
+export const sendMail = ({ to, subject, text, html }) => {
+  return transporter.sendMail({
+    from: `Auth API <${process.env.SMTP_USER}>`,
+    to,
+    subject,
+    text: text ?? '',
+    html,
+  });
```

Comment:
It would be beneficial to add error handling for the sendMail function to catch and log any errors that occur during the email sending process.